mp/src/game/client/momentum/ui/menus/hud_menu_static_system.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+class CHudMenuStaticSystem : public CAutoGameSystem
+{
+  public:
+    CHudMenuStaticSystem();
+
+    static void ShowMenu(const char *pszFileName);
+    static void HideMenu();
+    static bool IsMenuDisplayed(const char *pszMenuName);
```

Comment:
These don't need to be static, you have a global accessor with `g_pHudMenuStaticSystem`